src/select/select.js

```diff
@@ -296,7 +303,7 @@ class Select extends React.Component<PropsT, StatelessStateT> {
     const {type, disabled, selectedOptions} = this.props;
     const multiple = this.isMultiple();
     return (
-      <React.Fragment>
+      <div role="list">
```

Comment:
With adding an additional element we need to add it to the overrides API to be replaceable

Reply:
added here #409